amp-story/templates/ampmars/index.html

```diff
@@ -0,0 +1,967 @@
+<!doctype html>
+<html amp lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>The Mars Cruise</title>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <!-- ## Setup -->
+    <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
+    <link rel="canonical" href="index.html">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Josefin+Sans:700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Tienne:700" rel="stylesheet">
+    <script type="application/ld+json">
+      {
+        "bookendVersion": "v1.0",
+        "@context": "http://schema.org",
+        "@type": "CreativeWork",
+        "headline": "The Mars Cruise",
+        "url": "https://www.example.com"
+      }
+    </script>
+    <style amp-custom>
+      :root {
+        --yellow: #ffd200;
+        --bl-0: rgba(0, 0, 0, 0);
+        --bl-1: rgba(0, 0, 0, 1);
+      }
+
+      amp-story {
+        color: #eaeaea;
+        font-weight: 300;
+        font-family: 'Roboto', sans-serif;
+      }
+
+      amp-story .horiz {
+        display: flex;
+        flex-direction: row;
+      }
+      amp-story .horiz::before,
+      amp-story .horiz::after {
+        content: "";
+        flex: 1 1;
+        border-bottom: 0.0625em solid #fff;
+        margin: auto;
+      }
+      amp-story .horiz::before {
+        margin-right: 2.3rem;
+      }
+      amp-story .horiz::after {
+        margin-left: 2.3rem;
+      }
+
+      amp-story h2 {
+        color: var(--yellow);
+        font-size: 6.8vw;
```

Comment:
Is it intentional that all of the vertical styles (font sizes, line heights, vertical margins, etc) are all dependent on `vw` instead of `vh`?  Won't this mess up sizing on very wide devices?

Reply:
I haven't seen any issues with using `vw`.  When I initially used `vh` I found that the font size would change between devices since they all had a similar widths (aspect-ratio-wise) but their heights changed quite a bit.